### ISSUE TITLE
ua-logout doesn't redirect to login route

### ISSUE THREAD

**Issue by vrxj81** [userapp-io/userapp-angular#5]
The README says that using ua-logout the session is ended and one is redirected to the login route, but the redirect doesn't happen.

**Comment by timothyej** [comment]
Yes, it should redirect to the login route. Are you using the latest version (1.7.0) from bower?

**Comment by vrxj81** [comment]
no, i used following as per the docs:
&lt;script src="https://app.userapp.io/js/userapp.client.js"&gt;&lt;/script&gt;
&lt;script src="https://app.userapp.io/js/angularjs.userapp.js"&gt;&lt;/script&gt;

I will use bower qs suggested

**Comment by timothyej** [comment]
We weren't able to push some of the latest fixes to the our cdn. However, these are now out. Did it work when you used bower?
